docs/2.develop/contracts/security/welcome.md

```diff
@@ -0,0 +1,25 @@
+---
+id: welcome
+title: Security
+---
+
+Here you will find information about important concepts in security when developing decentralized applications.
+
+## [Smart Contract Audit Program](https://airtable.com/shrqu32NXPKjFYsrv)
+Pagoda’s [Smart Contract Audit Program](https://airtable.com/shrqu32NXPKjFYsrv) is an important step in expanding smart contract audits and enhancing ecosystem security. It allows projects to get an audit with a vetted firm in a timely manner. 
+
+Pagoda works with selected auditors and has experienced  firms on a retainer with pre-paid slots allowing key projects to get in queue faster than if they were to seek an audit independently, as the wait can be many months in most cases. 
+
+By filling out the [Smart Contract Audit Request form](https://airtable.com/shrqu32NXPKjFYsrv), projects building on NEAR can enter a queue for an audit. 
+
+We do request that projects provide clarity on when they will be ready to code freeze, and have their code completed to be able to get accurate scoping accomplished outlining exact timeline and cost. The projects also must have a source of funding before applying, either from the project itself or through a grant, and allow invoicing before the start date of the audit. 
```

Comment:
is this the same content as the text on `audits.md`? 

Should we just write here a short summary and link to `audits.md`?